src/controller/FileBrowserController.vala

```diff
@@ -28,7 +28,7 @@ namespace BrickManager {
     public class FileBrowserController : Object, IBrickManagerModule {
         const string PARENT_DIRECTORY_TEXT = "../";
         const string INITAL_DIRECTORY = "/home/robot";
-        const string USER_NAME = "ev3dev";
+        const string USER_NAME = "robot";
```

Comment:
Actually we do have a ev3dev group now (which the robot user is a member of). But the ttys are owned by root/tty and robot is not in the tty group. So we still need this code.
I'd suggest to first fix it and if we want to change it open a ticket and describing the desired way of work.